### PR TITLE
Add lepidopter release versioning script, rename Changelog

### LINKED ISSUE #58: Support identifying the version of a lepidopter image
Currently the lepidopter image ships with a file called `/etc/default/lepidopter` that contains some version information, though it appears to only mention the label of the version (ex. `LEPIDOPTER_BUILD="alpha"`) and not the actual full version string.

This should be a problem if we decide to make some future major releases that are also going to be in alpha.

I would also advise we change the format of the version to be something that is easier to be parsed (for example just including the raw version number) and perhaps place it inside of a more standard file (such as `/etc/lepidopter_version` similarly to how debian does it).

For backward compatibility reasons I would say we keep both files `/etc/default/lepidopter` being the variable and `/etc/lepidopter_version` being the raw version number.

### PR DESCRIPTION
**Pull request by anadahz**
This fixes #58.

### CODE REVIEW

File: scripts/release [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# git tag -a vX.X.X -m "Release msg" -s -u <keyid>"
+
+echo "Using tag reference: $(git tag -n |tail -n1) as release tag."
+
+NEW_RELEASE_VER=$(git describe --abbrev=0)
+
+echo ${NEW_RELEASE_VER} > ${VERSION_FILE}
+
+mv ChangeLog.md ChangeLog.md.old
+echo "#Lepidopter ${NEW_RELEASE_VER} [`date +%Y-%m-%d`]" > ChangeLog.md
```
Comment (hellais) [inline scripts/release]: It seems like the changelog is actually a `rst` file.

File: scripts/release [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# This script updates lepidopter version and ChangeLog files
+set -e
+
+VERSION_FILE="lepidopter-fh/etc/lepidopter_version"
+
+# First issue a release
+# git tag -a vX.X.X -m "Release msg" -s -u <keyid>"
```
Comment (hellais) [inline scripts/release]: You actually need to create the tag after you have updated the changelog.

**Comment by hellais** [comment]
Are you sure a script is needed for doing this process? It seems like this is a fairly manual process that requires some level of human intelligence and I am not sure what benefit there is in automating it since at the end of the day you just need to update 1 file.

File: scripts/release (hunk 1 above)
Comment (anadahz) [reply scripts/release]: The file Changelog.rst has been renamed to ChangeLog.md already in https://github.com/TheTorProject/lepidopter/pull/71/files

File: scripts/release (hunk 2 above)
Comment (anadahz) [reply scripts/release]: Thanks changed the order.

**Comment by anadahz** [comment]
I wouldn't have made this script just for ChangeLog, though it is quite important to have a consistency in `lepidopter_version` file that can be mistakenly set to a wrong version.

**Comment by hellais** [comment]
It seems like versioning information used to be stored inside of `/etc/default/lepidopter` (https://github.com/TheTorProject/lepidopter/blob/master/lepidopter-vmdebootstrap_build.sh#L5), while now it's being written to `/etc/lepidopter_version`. I would update this to write it to the same place and maintain consistency of the location and format of versioning information.

**Comment by anadahz** [comment]
These files serve different purposes: `/etc/default/lepidopter` stores a variable (`LEPIDOPTER_BUILD`) of the `git describe --tags` upon image build and `/etc/lepidopter_version` contains only the git annotated tag object.

Similar to `/etc/os-release` and `/etc/debian_version` in Debian.

**Comment by hellais** [comment]
When would this file be updated? Is it only updated when a person installs by burning a new image or should it also updated by the updater?

**Comment by anadahz** [comment]
The updater should update the `/etc/lepidopter_version` file and should add a variable to `/etc/default/lepidopter` in order to know that the specific lepidopter has been updated by the updater. I suggest not changing the `LEPIDOPTER_BUILD` variable as this will help us to know the specific image build version of lepidopter and can help us alleviate future issues that may arise.

**Comment by anadahz** [comment]
I'm going to proceed by merging this PR since it's what I'm using to build the test images.

**Pull request merged**